code/src/components/Album.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import Icons from './Icons.js'
+
+const Album = (props) => {
```

Comment:
Great solution to add all album and artist data in one component. It makes the code easy to read and follow.